localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -154,6 +154,9 @@ class EKFLocalizer : public rclcpp::Node
 
   double ekf_rate_;
   double ekf_dt_;
+  double pitch_from_ndt_;
+  double pitch_rate_;
+  rclcpp::Time t_receive_pose;
```

Comment:
Always add underscore after the member variables, it may confuse other developers.
```suggestion
  rclcpp::Time t_receive_pose_;
```

Reply:
> @SaltUhey Could you please add an explanation about this functionality to [the README of the ekf_localizer](https://github.com/autowarefoundation/autoware.universe/blob/main/localization/ekf_localizer/README.md)?

OK. I will add an explanation about this functionality to [the README of the ekf_localizer](https://github.com/autowarefoundation/autoware.universe/blob/main/localization/ekf_localizer/README.md).
Do I provide an additional explanation in [Functions](https://github.com/autowarefoundation/autoware.universe/blob/main/localization/ekf_localizer/README.md#functions)?